mobile/src/main/java/org/openhab/habdroid/model/CloudNotification.kt

```diff
@@ -21,17 +23,40 @@ import java.util.TimeZone
 import kotlinx.parcelize.Parcelize
 import org.json.JSONException
 import org.json.JSONObject
+import org.openhab.habdroid.core.connection.Connection
+import org.openhab.habdroid.util.IconBackground
+import org.openhab.habdroid.util.ImageConversionPolicy
+import org.openhab.habdroid.util.getIconFallbackColor
+import org.openhab.habdroid.util.map
 import org.openhab.habdroid.util.optStringOrNull
 
 @Parcelize
 data class CloudNotification internal constructor(
     val id: String,
+    val title: String,
     val message: String,
     val createdTimestamp: Long,
     val icon: IconResource?,
-    val severity: String?
+    val severity: String?,
+    val actions: List<CloudNotificationAction>?,
+    val onClickAction: CloudNotificationAction?,
+    val mediaAttachmentUrl: String?
 ) : Parcelable {
     val idHash get() = id.hashCode()
+
+    suspend fun loadImage(connection: Connection, context: Context, size: Int): Bitmap? {
+        mediaAttachmentUrl ?: return null
+        //if (mediaAttachmentUrl.startsWith("item:")) {
+        //    val itemName = mediaAttachmentUrl.removePrefix("item:")
+        //    val item = ItemClient.loadItem(connection, itemName)
+        //
+        //}
+        val fallbackColor = context.getIconFallbackColor(IconBackground.APP_THEME)
+        return connection.httpClient
```

Comment:
I know this is still in an early draft,  but i though i would mention that if the user puts in a relative path for media, so say `/static/foo.png` the client should try hitting that on the user's openhab (through local or remote settings).    I just made this change to IOS, and there is a PR for the cloud docs up right now to document it.   So the prefixes possible  are "http", "/" and "item" 👍   Other wise no other changes to what we discussed.

Reply:
The function https://github.com/openhab/openhab-android/blob/main/mobile/src/main/java/org/openhab/habdroid/util/HttpClient.kt#L94 is called internally by httpClient.get(), so relative paths should already work.

Reply:
Thats awesome.  Thanks!